src/renderer/screens/manager/index.js

```diff
@@ -1,21 +1,8 @@
 // @flow
 import React from "react";
-import styled from "styled-components";
 import ManagerConnect from "~/renderer/components/ManagerConnect";
 import Dashboard from "~/renderer/screens/manager/Dashboard";
 
-/* FIXME not fan of the Default Main idea that force the padding top... we don't want it here */
-const CompensateAlignment: any = styled.div`
```

Comment:
as discussed we'll see how to improve the global padding we have to not have to compensate here but still guarantee the entering screen is valigned.